zhmc_prometheus_exporter/zhmc_prometheus_exporter.py

```diff
@@ -95,6 +97,11 @@
     'other': ('localhost', 514),  # used if no key matches
 }
 
+# Sleep time and hysteresis in property fetch thread
+INITIAL_FETCH_SLEEP_TIME = 30
```

Comment:
INITIAL_FETCH_SLEEP_TIME is used both for the initial time and for the minimum time. It would be cleaner to have two separate constants, though.